releasenotes/notes/process-agent-status-command-6ea4385bea90836f.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
```

Comment:
Nit: you can generally drop this comment.
```suggestion
```

Reply:
Oh thanks, I forgot to remove that